artifacts/camara-cloudevents/event-subscription-template.yaml

```diff
@@ -864,17 +882,18 @@ components:
           schema:
             $ref: "#/components/schemas/ErrorInfo"
           examples:
-            PermissionDenied:
+            GENERIC_403_PERMISSION_DENIED:
+              description: Permission denied. OAuth2 token access does not have the required scope or when the user fails operational security
               value:
                 status: 403
-                code: "PERMISSION_DENIED"
-                message: "Client does not have sufficient permissions to perform this action"
-            TokenMismatch:
+                code: PERMISSION_DENIED
+                message: Client does not have sufficient permissions to perform this action.
+            GENERIC_403_SUBSCRIPTION_MISMATCH:
```

Comment:
I think SUBSCRIPTION_MISMATCH doesn't belong to Generic403, it's subscription specific
Can't we have only 1 403 type like SubscriptionPermissionDenied with 3 examples (permissiondenied, invalidtoken, subscription mismatch), and remove Generic403?

Reply:
@akoshunyadi  I used `CreateSubscriptionPermissionDenied403` only for the `POST /subscriptions`  and I have `403_INVALID_TOKEN_CONTEXT` specific for the POST
`Generic403` is for the GET & DELETE and here we have the `GENERIC_403_SUBSCRIPTION_MISMATCH` which applies only for GET & DELETE and not POST

Reply:
For me GenericXXX are types which we could put into/take from the common yaml and use for all APIs, because they don't contain any API specific hints. So I think we can have multiple 403 types here but if *Generic then it should only have generic examples.

Reply:
humm...hope i was able to follow you
I've defined `SubscriptionPermissionDenied403` for all subscription related 403 for the operations and I kept `Generic403` as response possible for the callback.